.travis.yml

```diff
@@ -2,15 +2,22 @@ language: python
 python:
     - 2.7
     - 2.6
+addons:
+    apt:
+        packages:
+            - oracle-java8-set-default
 install:
     - source utils/init-environment
     - pip install codecov
-services:
-    - elasticsearch
-# This is required for elasticsearch
+
+# Sleep is required for elasticsearch.
 before_script:
     - sleep 10
-script: coverage run --source shelf/ -m unittest discover -p *_test.py tests -b
+    - curl -v http://localhost:9200/
```

Comment:
Yes, also if Elasticsearch doesn't come up it just fails here. Probably not necessary but makes troubleshooting easier.